contracts/Core/StakeManager.sol

```diff
@@ -328,15 +327,17 @@ contract StakeManager is Utils, ACL, StakeStorage {
 
         uint256[] memory lowerCutoffsLastEpoch = _block.lowerCutoffs;
         uint256[] memory higherCutoffsLastEpoch = _block.higherCutoffs;
-        uint256[] memory mediansLastEpoch = _block.medians;
+        // Commented, because it was not being used, why was it defined if not being used?
+        // uint256[] memory mediansLastEpoch = _block.medians;
```

Comment:
We stop using `mediansLastEpoch` & `medianLastEpoch`, Any idea @hrishikeshio ?

P.S. It was detected by solhint.